docs/configuration/boottidal.md

```diff
@@ -8,23 +8,28 @@ Everytime you start Tidal, the software is reading from a configuration file usu
 Some users went really far into customizing their setup: [Jarmlib](https://github.com/jarmitage/jarmlib). You can take a look at their work to see how to extend your configuration file.
 
 As an example, here is the *vanilla* `BootTidal.hs` file used by the [Pulsar Package for Tidal](https://raw.githubusercontent.com/tidalcycles/pulsar-tidalcycles/master/lib/BootTidal.hs):
```

Comment:
This should link (at this point in time) to:
https://github.com/tidalcycles/Tidal/blob/1.9-dev/BootTidal.hs

If/when the main branch gets released, it will need an update